docs/modules/user-guide/partials/proc_migrating-components.adoc

```diff
@@ -2,14 +2,12 @@
 = Migrating components
 
 [role="_abstract"]
-This section describes how to migrate components to devfile v2. There ARE major changes in the `components` section of the devfile. A component specified in a v1.0 devfile will NOT work as it is in a devfile v2.0.
+This section describes how to migrate components to v2.0 devfiles. A component specified in a v1.0 devfile no longer works in a v2.0 devfile.
```

Comment:
I like some parts of the original descriptions where it describes there are changes in the components. The current suggested change sounds like the component does not work in general in a devfile 2.0 at all. This message should be more in the tone of changes that are needed before getting it to work again.